pkg/rpcservice/version.go

```diff
@@ -0,0 +1,13 @@
+package rpcservice
+
+import (
+	"context"
+
+	"github.com/stripe/stripe-cli/pkg/version"
+	"github.com/stripe/stripe-cli/rpc"
+)
+
+// Version returns the version of the Stripe CLI
+func (srv *RPCService) Version(ctx context.Context, req *rpc.VersionRequest) (*rpc.VersionResponse, error) {
+	return &rpc.VersionResponse{Version: version.Version}, nil
```

Comment:
you might want to add things like the git commit sha here

Reply:
I think I could add this field in a future PR.